### PR TITLE
#635 Operator Accounts print utility

### PR DESCRIPTION
**Pull request by otselnik**
Executed with:
`python3 -m proxy.accounts_info`
Output:
```json
{
   "accounts":[
      {
         "solana_address":"BMp6gEnveANdvSvspESJUrNczuHz1GF5UQKjVLCkAZih",
         "solana_balance":4979911815560,
         "neon_address":"0xd1b27c35c47bd37398ad6ab642b1bf63aec0ca43",
         "neon_balance":0,
         "neon_private":"0xa1f74239cbbc8dec7c7bc8c0ff17a12274ca46b6b05ec344b9203d2acb39f5be",
         "resource_keys":[
            {
               "address":"62MMKXigkapYUDBo1swJRZvyQjH37WPh8SRvpBD5WS4F",
               "status":"uninitialized"
            },
        ],
     }
  ]
}
````

### CODE REVIEW

**Review by afalaleev — CHANGES_REQUESTED** [review]
Looks good! But:
- In JSON add:
--- balance of storage/holder accounts
--- solana private
--- total-balance in SOL available for tx processing
--- total-balance in SOL on storage/holder accounts
--- total-balance in NEON 
- Move this functionality to proxy-cli utility as a command 
- In proxy-cli utility add simple commands:
--- to print list of  `solana-address  solana-balance solana-strorage/holder-balance` + total balance
--- to print list of `neon-address neon-balance` +total balance 
--- to print list of `neon-private-key` (can be used to add account to metamask)
--- to print list of `solana-private-key` (can be used to add account to metamask)
--- to print list of `storage-accounts`
--- to print list of `holder-accounts`

File: proxy-cli/__main__.py [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
 from .account import AccountHandler
 from .operator import OperatorHandler
+from .info import InfoHandler
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Client command line utility for Neon Proxy.')
     subparsers = parser.add_subparsers(title='command', dest='command', description='valid commands')
 
     account_handler = AccountHandler.init_args_parser(subparsers)
     operator_handler = OperatorHandler.init_args_parser(subparsers)
```
Comment (afalaleev) [inline proxy-cli/__main__.py]: Remove operator_handler
It was created for info_handler.

Comment (afalaleev) [inline proxy-cli/__main__.py]: Remove operator_handler
It was created for info_handler.

File: proxy-cli/info.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        elif args.subcommand == 'solana-private-key':
+            ret_js = self._solana_private_key_info(args)
+        elif args.subcommand == 'neon-private-key':
+            ret_js = self._neon_private_key_info(args)
+        elif args.subcommand == 'neon-address':
+            ret_js = self._neon_address_info(args)
+        elif args.subcommand == 'solana-accounts':
+            ret_js = self._solana_accounts_info(args)
+        elif args.subcommand == None:
+            ret_js = self._all_info(args)
```
Comment (afalaleev) [inline proxy-cli/info.py:63]: Only this case should print full json.
Other commands should print simple lines

File: proxy-cli/__main__.py (hunk 1 above)
Comment (otselnik) [reply proxy-cli/__main__.py]: done

File: proxy-cli/info.py (hunk 2 above)
Comment (otselnik) [reply proxy-cli/info.py:63]: fixed

**Review by afalaleev — DISMISSED** [review]

**Review by afalaleev — APPROVED** [review]

**Pull request merged**